tests/php/test-amp-facebook-embed-handler.php

```diff
@@ -14,11 +15,17 @@
  */
 class AMP_Facebook_Embed_Handler_Test extends WP_UnitTestCase {
 
+	use MarkupComparison;
+	use WithoutBlockPreRendering {
+		setUp as public prevent_block_pre_render;
+	}
+
 	/**
 	 * Set up.
 	 */
 	public function setUp() {
 		parent::setUp();
+		$this->prevent_block_pre_render();
```

Comment:
This calls `parent::setUp()` twice, once here, and once in `WithoutBlockPreRendering ::setUp` (which is mapped to `$this->prevent_block_pre_render();` here).